eZ/Bundle/EzPublishCoreBundle/Translation/Collector.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * File containing the Collector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Translation;
+
+/**
+ * Retrieves all installed ezplatform translation files.
+ */
+class Collector
```

Comment:
@bdunogier I renamed it to GlobCollector as suggested.

For the pattern, initially we decide to keep it simple for this first step so I suggest we stay on that.

I guess it's ok now ?